src/networking/APNSNetworkService.php

```diff
@@ -82,7 +74,6 @@ public function enqueueRequest( string $url, array $headers, string $body ): voi
 		curl_setopt( $ch, CURLOPT_VERBOSE, $this->debug );
 		curl_setopt( $ch, CURLOPT_PORT, $this->port );
 		curl_setopt( $ch, CURLOPT_CAINFO, $this->certificate_bundle_path );
-		curl_setopt( $ch, CURLOPT_SSL_VERIFYPEER, $this->ssl_verification_enabled );
```

Comment:
Checked on [libcurl](https://curl.haxx.se/libcurl/c/CURLOPT_SSL_VERIFYPEER.html) and it says that the default is true.

Reading [between the lines of the docs](https://www.php.net/manual/en/book.curl.php), I'm pretty sure PHP uses libcurl under the hood.